crates/client/derive/fuzz_test_executor/src/lib.rs

```diff
@@ -13,6 +13,12 @@ pub fn fuzz_test_executor(input: TokenStream) -> TokenStream {
                 let variant_name = &variant.ident;
                 quote! {
                     #enum_name::#variant_name (ix) => {
+                        #[cfg(fuzzing_with_stats)]
+                        let mut stats_logger = FuzzingStatistics::new();
+                        #[cfg(fuzzing_with_stats)]
```

Comment:
I dont think it is possible because if I put it in the block like this , then this part:

```
#[cfg(fuzzing_with_stats)]
stats_logger.increase_successfully_invoked(self.to_context_string());
```

will not "see it"

Reply:
I tried it and it seems to be working. It is the same feature flag so I don't see a reason why it should not work.

Reply:
Btw do we need the `increase_invoked` method? I think it would be better to increment the `invoked` counter directly within the `increase_successful` or `increase_failed` methods. This will save at least one `entry and modify`...